### ISSUE TITLE
Home Page

### ISSUE THREAD

**Issue by gkiplangat** [gkiplangat/adultyng#9]
Implement Home Page as in the design below;
![image](https://github.com/gkiplangat/adultyng/assets/56033401/e4e3aa63-40f9-4939-a158-50dae8376f19)

your  files should be named  as
```
index.html
home.css
main.js :incase you have one
images/: images folder
```
neccessary images and audio share on email

**Comment by glearnhub** [comment]
noted, will get back in case of anything

**Comment by gkiplangat** [comment]
you did not change the linking between the `index.html` and `main.js` after renaming if from `script.js` 

Check `line 180`
![image](https://github.com/gkiplangat/adultyng/assets/56033401/49d4abbe-139a-43db-87cb-ebd3081cdeb5)

**Comment by glearnhub** [comment]
> you did not change the linking between the `index.html` and `main.js` after renaming if from `script.js`
> 
> Check `line 180` ![image](https://user-images.githubusercontent.com/56033401/244936351-49d4abbe-139a-43db-87cb-ebd3081cdeb5.png)


I have corrected this with this commit : 85bb24f3867238fd8e7f0f67d184d7187b7f5b44

**Comment by gkiplangat** [comment]
Title of the page
the page title is `Gideon Restaurant` check on last line of the attached code snippet
```ts
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Gideon Restaurant</title>
```

**Comment by glearnhub** [comment]
> Title of the page the page title is `Gideon Restaurant` check on last line of the attached code snippet
> 
> ```ts
> <meta charset="UTF-8">
> <meta name="viewport" content="width=device-width, initial-scale=1.0">
> <title>Gideon Restaurant</title>
> ```

fixed it here: d972d8ec781c506d08ac356f8e02e4e14a209ffe

**Comment by gkiplangat** [comment]
well.
